.travis.yml

```diff
@@ -0,0 +1,32 @@
+---
+language: node_js
+node_js:
+  - "8"
+
+sudo: false
+dist: trusty
```

Comment:
trusty is default now, and `sudo: false` is deprecated

Reply:
I used the current .travis.yml file from Ember as a prototype of this 🙈 https://github.com/ember-cli/ember-new-output/blob/stable/.travis.yml#L22 I assume we should be adding this feedback there too?

Reply:
yeah, I think a PR for that is still missing 🙈